AUTHORS

```diff
@@ -3,12 +3,14 @@ Asuki Kono
 belono
 Christoph Heuel
 Cody (Quantified Code Bot)
+csoft2k
 Curtis // mashedkeyboard
 Davis Goglin
 Dean Rispin
 Dmytro Katyukha
 Hark
 Joel Lehtonen
+K
```

Comment:
I feel I might need some help to achieve this. Should I upload that file or thee is a command to perform that clustering?